eng/pipelines/pipeline-resources.yml

```diff
@@ -18,15 +18,22 @@ extends:
 
     containers:
       linux_x64:
-        image: mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7
+        image: mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-net8.0-cross-amd64
+        env:
+          ROOTFS_DIR: /crossrootfs/x64
+
+      linux_x86:
```

Comment:
Is this just for completeness's sake?

Reply:
Yes. I saw it in the runtime images and added for the future.